readme.md

```diff
@@ -0,0 +1,105 @@
+# Intro
+
+This is MVP for one of our client (UTSEUS) which helps teams on hackathons exchange skills among team members and pay for their help.  It has three modules: web client (React), server (NodeJS) and storage with some business logic on blockchain (ethereum). This doc descriebs how to setup environment and run server backed by blockchian. 
+
+## Install environment
+
+A whole environment is described in scripts of package.json. Almost all what you need is run this scripts from the server folder. 
+
+```
+npm run install
+npm run testrpc
+npm run truffle
+# run server API tests
+npm run test-mocha
+# run unit tests
+npm run test-jest
+```
+
+## Configure truffle.js
+
+Use one of accounts from output of testrpc to put into this truffle.js configuration 
+
+```
+require('babel-register')
+require('babel-polyfill')
+
+module.exports = {
+  networks: {
+    development: {
+      host: "localhost", 
+      port: 8545,
+      network_id: "*",
+      from: "0x041599451d9bac5bee252a5db569856085666f62",
+      gas: 4712388
+    }
+  }
+}
+```
+## Setup users
+Both skills and users are preconfigured on the server. To setup users go to `utils.js` and put new data into array `users` 
+
+## Setup skills
+To setup users go to file `dataset.js` and fill data into variable `skills`
+
+## Start the server
+Move back to the server folder and execute this command
+
+```
+npm run start
+```
+
+## API
+
+/users - to get list of users:
+```
+curl -X GET http://localhost:3000/api/v1/users
+
+{"users":[{"user":"lawlietyumiao"}]}
+```
+
+/skills - to get list of skills:
+
+```
+curl -X GET http://localhost:3000/api/v1/skills
+
+[{"skill":"business model"},{"skill":"financial planning"}]
+```
+
+/help/:mentee/:skill/:time - to register user with skill
+
+```
+curl -X POST http://localhost:3000/api/v1/register/Ricky/React
```

Comment:
shouldn't the list of skills also update then?

```
rngadam@rngadam-mac ~/c/s/s/server> curl -X POST http://localhost:3000/api/v1/register/Ricky/React
user was successfully registered⏎                                                                                                                        rngadam@rngadam-mac ~/c/s/s/server> curl -X GET http://localhost:3000/api/v1/skills
[{"skill":"business model"},{"skill":"financial planning"},{"skill":"brand identity"},{"skill":"mobile marketing"},{"skill":"graphic design"},{"skill":"UX design"},{"skill":"machine learning"},{"skill":"data harvesting"},{"skill":"data analysis"},{"skill":"data visualization"},{"skill":"software design"},{"skill":"html5"},{"skill":"agile management (SCRUM)"},{"skill":"lean management (KANBAN)"}]⏎   
```

although just registered React for Ricky, React doesn't appear in the list.

Reply:
You are right. I have to bring your attention to the case we deliver this project in very short time schedule to meet with business goals (I don't even talk about budgets). There are some shortcuts like:
- no validation on most of input parameters
- inconsistency between test and relevant data

For hackathon business use case, users operate with server only over the web client. They are able to provide only skill which they can found in the response on request to /skills. 

Despite on your feedback is fair, I propose to omit it for this version of project